app/src/components/programCard.jsx

```diff
@@ -97,10 +109,9 @@ const SeeMore = styled.a`
 `;
 
 const ToogleText = styled.span`
```

Comment:
```suggestion
const ToggleText = styled.span`
```

Ça vient pas de toi, mais quitte à revenir sur ce fichier, autant corriger la faute de frappe ^^
(Et toutes ses occurrences dans le fichier)
P.S. : j'ai candidaté pour cette [mission-ci](https://www.welcometothejungle.com/fr/companies/communaute-beta-gouv/jobs/developpeur-euse-fullstack-experimente-e-pour-la-plateforme-snu_paris?q=36c821504a0b4c93ceecf9875b078e55&o=49345c87-0314-4aec-87cf-a37f06e0d7d3) via ce [formulaire-ci](https://tally.so/r/nrD112), et j'attends toujours une date d'entretien 😉

Reply:
Bien vu !